addons/calendar/static/src/scss/calendar.scss

```diff
@@ -67,3 +67,7 @@
         font-weight: bold;
     }
 }
+.o_notification_manager, .o_notification {
```

Comment:
You probably don't want to do that: here you change the design for everyone. So either this is exactly what you want to do, and it should be in web, or you want to customize a calendar notif, and the rules should be properly scoped.